Dockerfiles/PostgreSQL/InitializationScripts/init.sh

```diff
@@ -0,0 +1,5 @@
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
```

Comment:
With Jeren's changes to the migrate functionality (Database is now created if none exist when running the migrate script) do we need this for the Postgres container? I could go either way

Reply:
If Jeren's script creates a database if one doesn't exist, then yes, this can go away. That was not the case originally. I'm going to pull latest from master, merge, and re-test

Reply:
Removed init script and updated docker-compose file to reflect that.